components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/main/java/org/wso2/carbon/identity/api/server/api/resource/v1/constants/APIResourceMgtEndpointConstants.java

```diff
@@ -82,7 +90,9 @@ public enum ErrorMessage {
                 "Both before and after parameters cannot be provided at the same time."),
         ERROR_CODE_SYSTEM_API_RESOURCE_NOT_MODIFIABLE("60012", "Cannot modify or delete System APIs.",
                 "Cannot modify or delete the read-only System APIs."),
-
+        ERROR_CODE_INVALID_REQ_ATTRIBUTES("60012", "Invalid attribute name.",
```

Comment:
The same error code is already used

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/508/commits/a554f52ff09da65532f69df35742292f62212fe4